src/unix/pty.cc

```diff
@@ -34,7 +34,16 @@
 #if defined(__GLIBC__) || defined(__CYGWIN__)
 #include <pty.h>
 #elif defined(__APPLE__) || defined(__OpenBSD__) || defined(__NetBSD__)
+/**
+ * From node v0.10.28 there is also a "util.h" in node/src, which
+ * confuses the compiler when looking for "util.h".
+ * without this compiles of electron 3.x in macOS fail.
+ */
+#if NODE_VERSION_AT_LEAST(8, 0, 0)
```

Comment:
Isn't Electron based on node 10?

Reply:
Yes it is, though as I understand it this is a catch all because the compiler cant find `#include <../include/util.h>` in some cases such as this, simply declaring `#include <util.h>` results in the issue.

I have now changed this to Node 10 with https://github.com/microsoft/node-pty/commit/4f18f21fc4284f6b7517df0380a93f6365b3f230, though. Success build at  https://travis-ci.org/the-j0k3r/node-pty-prebuilt-multiarch/jobs/569183066

The note above the change is the original removed in #322, so the node version here isnt linked to any specific electron build, its just to ensure the compiler finds include its looking for, Anywho changed to 10. =)